src/vmm/src/rpc_interface.rs

```diff
@@ -628,6 +635,11 @@ impl RuntimeApiController {
         create_snapshot(&mut locked_vmm, create_params, VERSION_MAP.clone())
             .map_err(VmmActionError::CreateSnapshot)?;
 
+        #[cfg(target_arch = "aarch64")]
+        {
+            warn!("This version of Firecracker is affected by a bug where the upper 64bit of the V0-V31 FL/SIMD registers are lost during snapshotting. This bug is fixed in Firecracker 1.1.4 and newer");
```

Comment:
Nit: let's have a const with this message so that we don't end up updating only one of these warnings by mistake in the future and end up having inconsistent messaging between Create and Load.

Reply:
done!